goty-server/src/test/kotlin/com/aleinin/goty/configuration/ZonedDateTimeDeserializerTest.kt

```diff
@@ -0,0 +1,79 @@
+package com.aleinin.goty.configuration
+
+import com.fasterxml.jackson.core.JsonParser
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.whenever
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeParseException
+
+@ExtendWith(MockitoExtension::class)
+internal class ZonedDateTimeDeserializerTest {
+    @Mock
+    lateinit var jsonParser: JsonParser
+
+    val deserializer = ZonedDateTimeDeserializer()
+
+    @Test
+    fun `Should accept supported ISO-8061 compliant strings`() {
```

Comment:
Fully supporting ISO-8601(just noticed that typo, will fix) is non-trivial and i don't think really worth it. ideally I can get away from custom handling entirely (see jackson thread) but if I can't I don't really think its worthwhile to try every conceivable 8601 format